aws-opentelemetry-distro/src/amazon/opentelemetry/distro/exporter/aws/metrics/console_emf_exporter.py

```diff
@@ -0,0 +1,107 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=no-self-use
+
+import logging
+import sys
+from typing import Any, Dict, Optional
+
+from opentelemetry.sdk.metrics.export import AggregationTemporality
+
+from .base_emf_exporter import BaseEmfExporter
+
+logger = logging.getLogger(__name__)
+
+
+class ConsoleEmfExporter(BaseEmfExporter):
+    """
+    OpenTelemetry metrics exporter for CloudWatch EMF format to console output.
+
+    This exporter converts OTel metrics into CloudWatch EMF logs and writes them
+    to standard output instead of sending to CloudWatch Logs. This is useful for
+    debugging, testing, or when you want to process EMF logs with other tools.
+
+    https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html
+
+    """
+
+    def __init__(
+        self,
+        namespace: str = "default",
+        preferred_temporality: Optional[Dict[type, AggregationTemporality]] = None,
+        preferred_aggregation: Optional[Dict[type, Any]] = None,
+        **kwargs: Any,
+    ) -> None:
+        """
+        Initialize the Console EMF exporter.
+
+        Args:
+            namespace: CloudWatch namespace for metrics (defaults to "default")
+            preferred_temporality: Optional dictionary mapping instrument types to aggregation temporality
+            preferred_aggregation: Optional dictionary mapping instrument types to preferred aggregation
+            **kwargs: Additional arguments (unused, kept for compatibility)
+        """
+        # No need to check for None since namespace has a default value
+        if namespace is None:
+            namespace = "default"
+        super().__init__(namespace, preferred_temporality, preferred_aggregation)
+
+    def _send_log_event(self, log_event: Dict[str, Any]) -> None:
```

Comment:
Since `_send_log_event` is more `aws_cloudwatch_emf_exporter` specific as we don't send LogEvent in console exporter, could this method name be made more generic? Maybe `_export`?